AppTaskQueue/appscale/taskqueue/queue.py

```diff
@@ -304,9 +299,7 @@ def add_task(self, task):
     except AttributeError:
       parameters['lease_expires'] = 0
 
-    result = self.db_access.session.execute(insert_task, parameters)[0]
-    if not result.applied:
-      raise InvalidTaskInfo('Task name already taken: {}'.format(task.id))
+    self._insert_task(parameters)
```

Comment:
@cdonati , I guess `retries` should be passed to the `_insert_task`. Shouldn't it?